src/main/java/hudson/plugins/mercurial/ChangeComparator.java

```diff
@@ -23,8 +23,13 @@
  */
 public abstract class ChangeComparator implements ExtensionPoint {
 	
-	public static ExtensionList<ChangeComparator> all() {
-        return Jenkins.getInstance().getExtensionList(ChangeComparator.class);
+    public static ExtensionList<ChangeComparator> all() {
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (jenkins != null) {
+            return jenkins.getExtensionList(ChangeComparator.class);
+        } else {
+            return ExtensionList.create((Jenkins)null, ChangeComparator.class);
```

Comment:
It's a common pattern  in Jenkins plugins and the core. IIRC the null branch was operational at some point, but I'm not sure about modern core versions